ledger/error/src/main/scala/com/daml/error/definitions/LedgerApiErrors.scala

```diff
@@ -91,211 +85,6 @@ object LedgerApiErrors extends LedgerApiErrorGroup {
         )
   }
 
-  @Explanation(
-    "Errors raised during the command execution phase of the command submission evaluation."
-  )
-  object CommandExecution extends ErrorGroup {
```

Comment:
Moving error groups like `CommandExecution` to separate, smaller files that are easier to manage.